utils/parsers.py

```diff
@@ -327,7 +327,7 @@ def _parse_typed_field(self, name, field_type, field_format):
             data_type = self._approximate_str_type(name)
 
         if data_type:
-            return BaseAPIField(data_type=data_type, name=name)
+            return BaseAPIField(data_type=data_type, name=name, definition=definition)
```

Comment:
As I mentioned above, I would explicitly pass in minimum and maximum here instead of the whole definition.